### PR TITLE
Closes AgileVentures/MetPlus_tracker#389

### PR DESCRIPTION
**Pull request by patmbolger**
## Story:

The job view page, for a job seeker, should have a control (button or link) that allows the JS to run a match process for this job against his/her résumé.

See initial mock-up for this in #350.

The match level (1.0 through 5.0) will be shown via graphical symbols - stars, for example - and that graphical representation will include depiction of match level that is not necessarily an integer (e.g. 3.8). This graphic representation has to be the same for the other "matching" stories.

### CODE REVIEW

File: app/controllers/jobs_controller.rb [hunk 1]
```diff
@@ -85,6 +86,8 @@ def create
   end
 
   def show
+    @resume = nil
+    @resume = pets_user.resumes[0] if pets_user.is_a?(JobSeeker)
```
Comment (joaopapereira) [inline app/controllers/jobs_controller.rb]: What happens if user do not have resume? Does this return nil?

File: app/controllers/jobs_controller.rb [hunk 2]
```diff
@@ -210,6 +213,28 @@ def revoke
...
+    raise 'Unsupported request' unless request.xhr?
+
+    job_seeker = JobSeeker.find(params[:job_seeker_id])
+    resume = job_seeker.resumes[0]
+
+    return render(json: { message: 'No résumé on file',
+                          status: 404 }) unless resume
+
+    result = ResumeCruncher.match_resume_and_job(resume.id, @job.id)
+    # result = ResumeCruncher.match_resume_and_job(1, 3) # << DEV PURPOSES
```
Comment (joaopapereira) [inline app/controllers/jobs_controller.rb]: We can remove comments

File: app/controllers/jobs_controller.rb (hunk 1 above)
Comment (patmbolger) [inline app/controllers/jobs_controller.rb]: Yes, this statement returns nil if no résumé.

**Comment by joaopapereira** [comment]
Please rebase and we can merge

**Comment by patmbolger** [comment]
Merge conflicts resolved.

**Pull request merged**